### DISCUSSION TITLE
Add build directory option to Gradle plugin

### DISCUSSION THREAD

**Discussion by csmuller** [neow3j/neow3j#326] [Ideas]
Simple question: How about we add a option to the Gradle plugin that allows the developer to define the output directory for the NEF, manifest and debug info?

**Comment by gsmachado** [comment]
@csmuller I fully support this idea.

Do you have a suggestion on how the property would be called?

**Reply by csmuller** [reply]
> @csmuller I fully support this idea.

Cool, i'll open an issue for it.

> Do you have a suggestion on how the property would be called?

Intuitively `outputDirectory`, but shortened to `outDir`.

**Comment by csmuller** [comment]
I added the issue #323 that covers this.

**Reply by gsmachado** [reply]
Cool! I can take this one.

Besides that, what do we do when a discussion topic comes to an "end"? Should we "lock conversation", should we "transfer discussion", or should we "delete discussion"? Or another action?

I think we could leave the discussion here since it could be useful to remind us about decisions...

**Reply by csmuller** [reply]
Locking the conversation simply restricts write access to us. It seems that there is no "close" option for discussions. I guess we'll just have to leave it as it is.
